spec/ace.md

```diff
@@ -23,7 +23,9 @@ This UUID is exposed to the pod through the [Metadata Service](#app-container-me
 
 #### Filesystem Setup
 
-Each app in a pod will start chrooted into its own unique read-write filesystem before execution.
+Each app in a pod will start chrooted into its own unique filesystem before execution.
+If the optional field `apps.readOnlyRootFS` is set to `true` in the ACI, 
```

Comment:
NACK, this is _not_ a field in the ACI - it's in the pod manifest.
"If the app's entry in the `PodManifest` has the field `readOnlyRootFS` is set to `true`, ..."